ImGuiApp/ImGuiController/ImGuiController.cs

```diff
@@ -147,6 +154,37 @@ private static void OnKeyDown(IKeyboard keyboard, Key keycode, int scancode) =>
 	private static void OnKeyUp(IKeyboard keyboard, Key keycode, int scancode) =>
 		OnKeyEvent(keyboard, keycode, scancode, down: false);
 
+	private static void onMouseScroll(IMouse mouse, ScrollWheel scroll)
```

Comment:
`onMouseScroll` should be `OnMouseScroll`

Reply:
Fixed.